crates/reporting/tests/test_reporting.rs

```diff
@@ -5645,11 +5645,16 @@ All branches in an `if` must have the same type!
     5│          1 -> True
                   ^^
 
-    The arrow -> is only used to define cases in a `when`.
+    The arrow -> is used to define cases in a `when` expression:
 
         when color is
             Red -> "stop!"
             Green -> "go!"
+
+    And defining functions:
```

Comment:
I think "and to define a function" sounds slightly better here. Then the whole thing reads as a sentence

> The arrow -> is used to define cases in a `when` expression, and to define a function:

also, what you're changing here is the test output. This message is constructed in `crates/reporting/src/error/parse.rs ` (~ line 280), so you'll have to change that too.

Reply:
Made this change. :+1: